app/src/main/res/drawable/ic_equalizer_black_24px.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/358f4d3d5d1451906a8d4af2536fd5c14fbe83c0/app/src/main/res/drawable/ic_equalizer_black_24px.xml#L1"></span>The resource <code>R.drawable.ic_equalizer_black_24px</code> appears to be unused</td>
    </tr>
  </tbody>
</table>